tests/benchmarks/object_footprint/testcase.yaml

```diff
@@ -1,55 +1,55 @@
 tests:
-  test_001:
+  benchmark.footprint.001:
```

Comment:
These changes don't look related to the script changes. Should they be in the "tests: fix test identifiers" commit instead?